fastlane/metadata/de-DE/subtitle.txt

```diff
@@ -1 +1 @@
-Erstelle deine neue Website
+Erstelle eine Website oder ein Blog
```

Comment:
Just a heads up that this will exceed the 30 char limit. (I remember you fixing it manually last time and asking translators for a fix at the source in some P2 or Slack, but apparently they didn't tackle it yet)

Reply:
Yep. This keeps happening. Your prompt was the push I needed to suggest these 30-characters-compliant translations into GlotPress myself. 🤞

Reply:
Cool. Let's merge it as is for now and hope the fixed copy gets accepted in GlotPress by the time you'll do the final release then 👍